common/src/main/java/org/entcore/common/http/filter/ResourceProviderFilter.java

```diff
@@ -63,11 +65,17 @@ protected void loadFiltersMapping() {
 					JsonObject filter = new JsonObject(line);
 					String method = filter.getString("method");
 					String f = filter.getString("filter");
+
+					String[] arguments = filter.getJsonArray("arguments") != null ?
+							filter.getJsonArray("arguments").stream()
```

Comment:
La partie "arguments" de la PR est un nice-to-have.

Perso, je trouve qu'elle laisse la champ ouvert à [mettre ici ce que tout développeur ingénieux peut inventer comme hack :rofl: ]
Bref, c'est une bonne idée mais l'implémentation en l'état me parait difficile à maintenir dans le temps.
C'est juste mon avis.

Reply:
On peut laisser en l'état, je garde ce point en tête pour la refonte.